wordpress-mobile/WordPress-iOS
# Update gutenberg version to 1.30.0

Update the Gutenberg version of the pod to 1.30.0

Gutenberg PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/2363

To test:
 - Check the Gutenberg editor by smoke testing the [writing flow here](https://github.com/wordpress-mobile/test-cases/tree/master/test-cases/gutenberg/writing-flow)
 - Check the new features added in this release:
   - Support for breaking out of captions/citation authors by pressing enter on the following blocks: image, video, gallery, quote, and pullquote.
   - Adds editor support for theme defined colors and theme defined gradients on cover and button blocks.

PR submission checklist:
